src/components/Boards/components/BoardTile/BoardTile.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import CardActions from '@material-ui/core/CardActions';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import CardHeader from '@material-ui/core/CardHeader';
+import Avatar from '@material-ui/core/Avatar';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import Card from '@material-ui/core/Card';
+
+interface Props {
+  id: string;
+  name: string;
+}
+
+export default function DefaultListItem(props: Props) {
+  return (
+    <Grid item xs={6} sm={3}>
```

Comment:
Question: How does it look good on larger screens?

Reply:
Looks ok -- little wide. We should probably wrap the container at some point. We can do polish later.

Reply:
Nice!